src/components/pages/admin/events/dashboard/fanNotifications/sections/ActionButton.js

```diff
@@ -74,7 +79,7 @@ class ActionButton extends Component {
 					Change
 				</Button>
 			);
-		} else if(isNotificationAfter) {
+		} else if(isNotificationAfter && !hasEventStarted) {
```

Comment:
This works currently because it's the negative of the wrong logic.
```suggestion
		} else if(isNotificationAfter && hasEventStarted) {
```